artifactory.py

```diff
@@ -1887,10 +1887,12 @@ def creator(self):
         """
         return self._accessor.creator(self)
 
-    def is_dir(self):
+    def is_dir(self, follow_symlinks=False):
```

Comment:
For consistency with pathlib/pathlib-abc:

```suggestion
    def is_dir(self, *, follow_symlinks=True):
```